models/model_runner.py

```diff
@@ -51,26 +68,25 @@ def execute_tracking_action():
 def execute_reid_action():
     """
     Using model_runner:
-    re-id-train --reid_config ./fast-reid/configs/DukeMTMC/bagtricks_R101-ibn.yml --model_weights
-    ./fast-reid/checkpoints/duke_bot_R101-ibn.pth --dataset "DukeMTMC-reID-test"
+    re-id-train --reid_config ./fast-reid/configs/DukeMTMC/bagtricks_R101-ibn.yml --device cuda:1 --reid_opts
+    MODEL.WEIGHTS ./fast-reid/checkpoints/duke_bot_R101-ibn.pth DATASETS.DATASET query-2.8_test-4.8
```

Comment:
Please add an example include all optional args, so we can re-run expirements with ease

Reply:
The changes I did are the updated experiment. The optional args for the reID model are every entrance in the reID config file, so there are dozens of them. On the other hand, the new `optional_args` from line 41 are relevant only for the tracking model, so I didn't add an example of this here.
Let me know if you understand what I mean because it is a bit confusing.